bin/run_mtdomegui

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# This file is part of ts_mtdomegui.
+#
+# Developed for the Vera Rubin Observatory Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+
```

Comment:
Please add these missing lines:
```
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Reply:
Added in the commit: [15564fe](https://github.com/lsst-ts/ts_mtdomegui/pull/1/commits/15564fef4c63a1980a78f362483b8ee50de8a05c). Thanks!